src/universal/conf/cassandra-env.sh

```diff
@@ -96,4 +96,7 @@ fi
 # to the location of the native libraries.
 JVM_OPTS="$JVM_OPTS -Djava.library.path=$CASSANDRA_HOME/lib/sigar-bin"
 
-JVM_OPTS="$JVM_OPTS $JVM_EXTRA_OPTS"
\ No newline at end of file
+JVM_OPTS="$JVM_OPTS $JVM_EXTRA_OPTS"
+
+# ConductR: Replace dead node when restarting a new Cassandra instance on the same address
+JVM_OPTS="$JVM_OPTS -Dcassandra.replace_address=$CAS_STORAGE_BIND_IP"
```

Comment:
@huntc Adding this JVM option solves the problem of restarting a cassandra instance on a node address where previously a cassandra instance was running, e.g.:

```
conduct run cassandra --scale 3
conduct run cassandra --scale 2
conduct run cassandra --scale 3
```

However, when using this JVM option for a fresh cassandra cluster, e.g.:

```
sandbox run 2.0.1
conduct load cassandra
conduct run cassandra
```

..then the Cassandra startup fails:

```

```

In other words, the `cassandra.replace_address` should only be added on demand, when a dead Cassandra node of an existing Cassandra cluster should be replaced. The option cannot be set for a fresh Cassandra cluster that needs to be formed.

I have no idea how to distinguish between these two different scenarios in the `cassandra-env.sh` script. We should only add this JVM option if a ConductR cluster has been formed and if a dead cassandra instance with the address `$CAS_STORAGE_BIND_IP` exists. While we could find out an existing Cassandra cluster with the `$CAS_STORAGE_OTHER_IPS` env, I don't know how to check if a Cassandra node is down or not. Generally, you can check that with the `nodetool` that connects a Cassandra cluster via JMX. However, we don't have JMX enabled and we also cannot expect that `nodetool` is installed on the machine.

Do you have another idea how to distinguish between these two scenarios in the `cassandra-env.sh` script?